pkg/component/autoscaling/vpa/updater.go

```diff
@@ -128,6 +140,31 @@ func (v *vpa) reconcileUpdaterClusterRoleBinding(clusterRoleBinding *rbacv1.Clus
 	}}
 }
 
+func (v *vpa) reconcileUpdaterClusterRoleInPlace(clusterRole *rbacv1.ClusterRole) {
+	clusterRole.Labels = getRoleLabel()
+	clusterRole.Rules = []rbacv1.PolicyRule{
+		{
+			APIGroups: []string{""},
+			Resources: []string{"pods", "pods/resize"},
+			Verbs:     []string{"patch"},
+		},
+	}
+}
+
+func (v *vpa) reconcileUpdaterClusterRoleBindingInPlace(clusterRoleBinding *rbacv1.ClusterRoleBinding, clusterRole *rbacv1.ClusterRole, serviceAccountName string) {
+	clusterRoleBinding.Labels = getRoleLabel()
+	clusterRoleBinding.Annotations = map[string]string{resourcesv1alpha1.DeleteOnInvalidUpdate: "true"}
+	clusterRoleBinding.RoleRef = rbacv1.RoleRef{
+		APIGroup: rbacv1.GroupName,
+		Kind:     "ClusterRole",
+		Name:     clusterRole.Name,
+	}
+	clusterRoleBinding.Subjects = []rbacv1.Subject{{
+		Kind:      rbacv1.ServiceAccountKind,
+		Name:      serviceAccountName,
+		Namespace: v.namespaceForApplicationClassResource(),
+	}}
+}
 func (v *vpa) reconcileUpdaterRoleLeaderLocking(role *rbacv1.Role) {
```

Comment:
You could add empty line between the funcs:

```suggestion
}

func (v *vpa) reconcileUpdaterRoleLeaderLocking(role *rbacv1.Role) {
```

Reply:
Thanks for catching this, added the missing line in my latest updates.